tests/fast-integration/eventing-test/README.md

```diff
@@ -28,14 +29,35 @@ COMPASS_CLIENT_SECRET=       # Only required if KYMA_TYPE=SKR
 COMPASS_TENANT=              # Only required if KYMA_TYPE=SKR
 GARDENER_KUBECONFIG=         # Only required if KYMA_TYPE=SKR
 ```
+>**IMPORTANT:** The `TEST_SUFFIX` is required for every test run for SKR cluster. It needs to be 4 characters long, as it is the name for the compass scenario.
+> The eventing tests are adding a scenario with the `"test-TEST_SUFFIX"` name and if you did not clean the compass scenario (which is done in the `eventing-test-cleanup.js` script),
```

Comment:
```suggestion
> The eventing tests add a scenario with the `"test-TEST_SUFFIX"` name. 
> You must clean the compass scenario, which is done in the `eventing-test-cleanup.js` script. If you did not clean it, the tests try to reuse an existing scenario and will fail during the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.
```

Trying to split that very long sentence. Also trying to avoid that a sentence is split across two lines.

Reply:
Updated. I actually even removed the second part of the second sentence, as it is covered in the troubleshooting part.